src/iperf_api.c

```diff
@@ -1908,7 +1923,9 @@ send_results(struct iperf_test *test)
 		}
 	    }
 	    if (r == 0 && test->debug) {
-		printf("send_results\n%s\n", cJSON_Print(j));
+                char *str = cJSON_Print(j);
+		printf("send_results\n%s\n", str);
+                free(str);
```

Comment:
same doubt of block 1758-1761

Reply:
Same comment from me about `cJSON_free()`.